### PR TITLE
fix: visual mode mapping use xmap if possible

### PR DESCRIPTION
**Pull request by tsuyoshicho**
Currently use vmap, map to select-mode.

This PR use xmap if possible, resolve it.

### CODE REVIEW

File: plugin/gf/user.vim [hunk 1]
```diff
@@ -52,12 +52,18 @@ call s:define_ui_key_mappings()
 
 command! -bang -bar -nargs=0 GfUserDefaultKeyMappings
 \ call s:cmd_GfUserDefaultKeyMappings(<bang>0)
+
+let s:mapcmd = ['nmap', 'vmap']
+if exists(':xmap') == 2
```
Comment (kana) [inline plugin/gf/user.vim:57]: This branching is verbose. `:xmap` is always available because vim-gf-user requires Vim 7.3 or later.

Comment (kana) [reply plugin/gf/user.vim:57]: Line 36 must be updated too for consistency.

**Review by kana — CHANGES_REQUESTED** [review]

File: plugin/gf/user.vim (hunk 1 above)
Comment (tsuyoshicho) [reply plugin/gf/user.vim:57]: Okay, This plugin's policy is...

- Vim : 7.3 or later -> xmap use OK.
- vmap using not need -> replace xmap OK.

I understood above.

next fix wait a little.

Comment (tsuyoshicho) [reply plugin/gf/user.vim]: Fix

**Review by kana — APPROVED** [review]

**Comment by tsuyoshicho** [comment]
Thanks you

**Pull request merged**